elasticsearch/sgconfig/sg_config.yml

```diff
@@ -3,8 +3,7 @@ searchguard:
     http:
       xff:
         enabled: true
-        remoteIpHeader: 'remoteAddress'
-        proxiesHeader: 'remoteAddress'
+        remoteIpHeader: 'x-forwarded-for'
```

Comment:
No.  Basically, until SG 2.4, the above settings were ignored, so SG was always using the default, which was `x-forwarded-for`.  See https://github.com/floragunncom/search-guard/commit/056d14a64b88fae2484108e285d024bedd4f2e71